chain/chain/src/store.rs

```diff
@@ -2812,10 +2810,20 @@ mod tests {
         assert!(chain.mut_store().get_next_block_hash(blocks[6].hash()).is_ok());
     }
 
-    /// Test that MAX_HEIGHTS_TO_CLEAR works properly
+    /// Test that `gc_blocks_limit` works properly
     #[cfg(feature = "expensive_tests")]
     #[test]
     fn test_clear_old_data_too_many_heights() {
+        for i in 1..10 {
+            println!("gc_blocks_limit == {:?}", i);
+            test_clear_old_data_too_many_heights_common(i);
+        }
+        test_clear_old_data_too_many_heights_common(25);
+        test_clear_old_data_too_many_heights_common(50);
+        test_clear_old_data_too_many_heights_common(87);
```

Comment:
How are these numbers chosen?

Reply:
arbitrary

Reply:
Can we use a random number generator?

Reply:
Yes we can, however this is pretty sanity test. If it passes for some numbers, it will pass for all other numbers as well.